### DISCUSSION TITLE
Tables and lists: Using & formatting them correctly

### DISCUSSION THREAD

**Discussion by daobrien** [StyleGuides/WritingStyleGuide#360] [Ideas]
https://chat.google.com/room/AAAAagA7kdc/cTEE1a5JqIA
A very long discussion, not only about tables and lists, but about layout, render in different modalities, and more. This discussion focuses on when we should use tables, when we should use lists, and the preferred way to build them for a good UX.
@bonnevil @dsacco15 @niklucas @flozano @t1s1 @nmuller66

**Comment by daobrien** [comment]
It would be good to know how far off the retirement of PDF files is, even roughly. If we knew that it would help the discussion because we could decide whether or not these PDF files will be a major contributing design factor.

E.g., what we can do in a table in ROLE right now we can't do in a PDF because it just won't fit (page width).

**Reply by niklucas** [reply]
> It would be good to know how far off retirement of PDF files is

PDFs are never going to go completely away. We will still use them as a print asset. However, will will be implementing the redesigned style that we are focusing primarily on the ePub form for the PDF as well. So stylistically it will be quite different.
